src/test/java/lotto/LottoTest.java

```diff
@@ -0,0 +1,89 @@
+package lotto;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+import java.util.List;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class LottoTest {
+
+  @DisplayName("로또를 생성한다")
+  @Test
+  void createLotto() {
+    Lotto lotto = new Lotto(List.of(1, 2, 3, 4, 5, 6));
```

Comment:
```suggestion
    Lotto lotto = new Lotto(1, 2, 3, 4, 5, 6);
```
가변 인자를 받는 생성자를 추가해 위와 같이 구현할 수 있도록 지원하면 Lotto를 사용하는 개발자 입장에서 편리성이 있지 않을까?